system/Test/CIDatabaseTestCase.php

```diff
@@ -135,6 +142,8 @@ public function __construct()
 	 */
 	public function setUp()
 	{
+		$this->loadDependencies();
```

Comment:
How does this operate now, then. I take it doesn't throw an exception, but throws up a lot of errors? If that's the case, I'm fine, since the docs clearly state that you need to setup a test connection or stop the tests from running it.